lark/parsers/lalr_parser.py

```diff
@@ -29,7 +30,7 @@ def deserialize(cls, data, memo, callbacks, debug=False):
         inst.parser = _Parser(inst._parse_table, callbacks, debug)
         return inst
 
-    def serialize(self, memo):
+    def serialize(self, memo = None) -> Dict[str, Any]:
```

Comment:
Adding `= None` here to make mypy happy, otherwise the subclass doesn't match the parent.

Reply:
It's probably more correct to say the return type is `Dict[Any, Any]`? But I'm rusty on that code, didn't touch it in a long time.

Reply:
No, `Dict[str, Any]` is correct, this is being fed into json.

Reply:
```suggestion
    def serialize(self, memo: Any = None) -> Dict[str, Any]:
```